index_template.html

```diff
@@ -10,7 +10,7 @@
     
 <h2>One shots and attitude errors</h2
 
-<h3>(Updated 
+<h3>(Updated
```

Comment:
And I meant to just write out the processing time from the jinja2 but the flastmod seems fine.  Need to put back the space.